huaweicloud/config/config.go

```diff
@@ -524,6 +533,36 @@ func (c *Config) getDomainID() (string, error) {
 	return all[0].ID, nil
 }
 
+func (c *Config) getUserIDbyName(name string) (string, error) {
+	identityClient, err := c.IdentityV3Client(c.Region)
+	if err != nil {
+		return "", fmt.Errorf("Error creating IAM client: %s", err)
+	}
+
+	opts := users.ListOpts{
+		Name: name,
+	}
+	allPages, err := users.List(identityClient, opts).AllPages()
+	if err != nil {
+		return "", fmt.Errorf("query IAM user %s failed, err=%s", name, err)
```

Comment:
suggest to using `fmtp.Errorf`, the content of `err` needs to be converted

Reply:
In config package, all error messages and log messages does not contain *Huaweicloud*, so it`s not necessary to ues fmtp.